packages/studio/src/components/RenderModal/SchemaEditor/create-zod-values.ts

```diff
@@ -17,7 +17,9 @@ export const createZodValues = (
 		case zodRuntime.ZodFirstPartyTypeKind.ZodString:
 			return '';
 		case zodRuntime.ZodFirstPartyTypeKind.ZodNumber:
-			return 0;
+			return (
+				(def as z.ZodNumberDef).checks.find((c) => c.kind === 'min')?.value ?? 0
+			);
```

Comment:
@JonnyBurger This fixes this particular issue with the `min()` or `gte()`. Would it make sense to also cover other checks like `gt()` and `positive()`, `multipleOf()`, etc...?

And then would it make sense to also extend the same method to other types like `string` and `BigInt` for example?

Reply:
I thought about it and the only one I can think of is `max && max.value < 0`.

Otherwise, all possible checks should pass with `0`.